java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/Server.java

```diff
@@ -858,6 +859,7 @@ protected LanguageClient client() {
     public static final String JAVA_SUPER_IMPLEMENTATION =  "java.super.implementation";
     public static final String JAVA_SOURCE_FOR =  "java.source.for";
     public static final String GRAALVM_PAUSE_SCRIPT =  "graalvm.pause.script";
+    public static final String DB_ADD_CONNECTION =  "micronaut.add.db.connection";
```

Comment:
Why is this command prefixed by `micronaut`? The DB explorer is completely micronaut unrelated, right?